contracts/most/Utmost.sol

```diff
@@ -0,0 +1,132 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "./GenesisUtmost.sol";
+import "./../consensus/CoconsensusModule.sol";
+import "../proxies/MasterCopyNonUpgradable.sol";
+import "./../utility-token/UtilityToken.sol";
+
+/**
+ * @title Utmost contract implements UtilityToken. Utmost is an ERC20 token
+ *        wrapper for the base coin that is used to pay for gas consumption on
+ *        the auxiliary chain.
+ */
+contract Utmost is MasterCopyNonUpgradable, GenesisUtmost, UtilityToken, CoconsensusModule {
+
+    /* Constants */
+
+    /** Token Symbol */
+    string public constant TOKEN_SYMBOL = "UM";
+
+    /** Token Name */
+    string public constant TOKEN_NAME = "Utmost";
```

Comment:
@jayeshbairagi , Please not this, we need to confirm what should be the symbol and name.